src/extension/geopkg-output/src/main/java/applicationContext.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+ Copyright (C) 2014 - Open Source Geospatial Foundation. All rights reserved.
+ This code is licensed under the GPL 2.0 license, available at the root
+ application directory.
+ -->
+<!DOCTYPE beans PUBLIC "-//SPRING//DTD BEAN//EN" "http://www.springframework.org/dtd/spring-beans.dtd">
+<beans>
+
+    <bean id="geopkgGetMapOutputFormat" class="org.geoserver.wms.geopackage.GeoPackageGetMapOutputFormat">
+      <constructor-arg ref="webMapService" />
+      <constructor-arg ref="wms" />
+      <constructor-arg ref="gwcFacade" />
+    </bean>
+    
+    <bean id="geopkgGetFeatureOutputFormat" class="org.geoserver.wfs.geopackage.GeoPackageGetFeatureOutputFormat">
+      <constructor-arg ref="geoServer" />
+    </bean>
+     
+    <bean id="geopkgDataStoreFactoryInitializer" class="org.geoserver.geopackage.GeoPkgDataStoreFactoryInitializer">
+        <property name="resourceLoader" ref="resourceLoader"/>
+    </bean>
+
+    <!-- bean so this will appear on the Modules list in the GS web interface-->
+    <bean id="GeoPKGOutputExtension"
```

Comment:
Thanks for including in module status list

Reply:
This self-review style is weird... or are you committing stuff on behalf of someone else? :D

Reply:
@davidblasby has been doing the work, with some assistance from me as needed.  I am catching up near the end and submitted the PR.